lms/envs/common.py

```diff
@@ -579,6 +579,7 @@
 ASSET_KEY_PATTERN = r'(?P<asset_key_string>(?:/?c4x(:/)?/[^/]+/[^/]+/[^/]+/[^@]+(?:@[^/]+)?)|(?:[^/]+))'
 USAGE_ID_PATTERN = r'(?P<usage_id>(?:i4x://?[^/]+/[^/]+/[^/]+/[^@]+(?:@[^/]+)?)|(?:[^/]+))'
 
+USERNAME_PATTERN = r'(?P<username>[\w.@+-]+)'
```

Comment:
USERNAME_PATTERN also exists in /edx-platform/lms/djangoapps/mobile_api/users/urls.py.

Can you clean that up so it also refers to this variable?